lib/ansible/cli/inventory.py

```diff
@@ -77,16 +77,18 @@ def init_parser(self):
         action_group.add_argument("--list", action="store_true", default=False, dest='list', help='Output all hosts info, works as inventory script')
         action_group.add_argument("--host", action="store", default=None, dest='host', help='Output specific host info, works as inventory script')
         action_group.add_argument("--graph", action="store_true", default=False, dest='graph',
-                                  help='create inventory graph, if supplying pattern it must be a valid group name')
+                                  help='Create inventory graph, if supplying pattern it must be a valid group name')
+        action_group.add_argument("--dot", action="store_true", default=False, dest='graph_dot',
```

Comment:
should we ask for a file name? (default to stdout is fine)

Reply:
I don't think it's relevant for this change as other "formats" options don't have such behavior.
I'd rather prepare another MR to implement a "-o" parameter and allow any of the options (graph, json, list, ...) to have an output file. wdyt ?

Reply:
disregard previous comment. --output is already implemented (silly me).
it just work (--output FILE, and if no output provided, default to stdout)